service/pure/src/pure.rs

```diff
@@ -30,23 +30,22 @@ use eyre::WrapErr;
 
 const CONFIG_FILE_PATH: &str = "/tmp/multiaddr_config";
 const DEFAULT_TIMEOUT_SEC: u64 = 1u64;
-const DEFAULT_LOCAL_API_MULTIADDR: &str = "/ip4/127.0.0.1/tcp/5001";
```

Comment:
I think `ipfs-adapter` should have sensible defaults so it works right after deploy